src/uast.cc

```diff
@@ -137,95 +244,155 @@ void *UastIteratorNext(UastIterator *iter) {
 
 NodeIface UastGetIface(const Uast *ctx) { return ctx->iface; }
 
+char *LastError(void) {
```

Comment:
Why has this been moved here?

Reply:
To keep all the `UastFilter*` functions consecutive in the file.

Reply:
Yes, but this was at the bottom of the file, it could have been left there (following the same order than the header file).